Turn/src/Store.cpp

```diff
@@ -3,13 +3,101 @@
 
 #include "..\include\Common.h"
 #include "..\include\Store.h"
+#include "..\include\ItemTypes.h"
 
 
 using namespace std;
 using namespace Common;
 
-void Store::StoreFront(Player* _Player){
-    ClearScreen();
-    cout << "Work in progress!" << endl;
-    system("PAUSE");
+const unsigned short arrowsCost = 200;
+const unsigned short bombCost = 100;
+const unsigned short potionCost = 150;
+const unsigned short whetstoneCost = 20;
+
+//Messages
+string lowCoinsMessage = " Insufficient Coins !";
```

Comment:
These strings should be constant

Reply:
should i move all the const values such as Cost and Messages to common.h and common.cpp ?